adserver/views.py

```diff
@@ -286,7 +286,9 @@ def get_context_data(self, **kwargs):
             {
                 "advertiser": self.advertiser,
                 "flight": self.flight,
-                "ad_types": AdType.objects.exclude(description="")[:5],
+                "ad_types": AdType.objects.exclude(deprecated=True).exclude(
+                    description=""
+                )[:5],
```

Comment:
Is there a reason to filter this by `:5`? Presumably we just want all of them?

Reply:
Well, this is 5 with descriptions. If we had say 10 ad types with descriptions, there would be a lot of text on the right side. It isn't wrong but it probably isn't what we want.